client/src/components/ratings-reviews/ReviewsList.jsx

```diff
@@ -10,9 +10,27 @@ const ReviewsList = () => {
   const sortedReviewsList = useSelector(state => state.sortedReviewsList);
   const reviewsCount = useSelector(state => state.reviewsCount);
   const starSort = useSelector(state => state.starSort);
+  const keywordSearch = useSelector(state => state.keywordSearch);
   const dispatch = useDispatch();
 
-  let reviewsList = sortedReviewsList ? sortedReviewsList.slice(0, reviewsCount) : fullReviewsList.slice(0, reviewsCount);
+  let tempReviewsList = sortedReviewsList ?? fullReviewsList;
+  if (keywordSearch) {
+    let result = [];
+    for (let review of tempReviewsList) {
+      if (review.body.toLowerCase().indexOf(keywordSearch.toLowerCase()) >= 0
```

Comment:
I think this will come up with false positives; for instance: if I searched for "hi", you'll get instances of "this" since "hi" is a substring of "this".

One solution could be to use RegExp
`review.body.toLowerCase().match(new RegExp("\\b" + keywordSearch.toLowerCase() + "\\b")) != null;`

This will check for individual words, rather than substrings.